package com.github.akhpkn.pdp.domain.task.exception

import com.github.akhpkn.pdp.exception.NotFoundException

class TaskNotFoundException : NotFoundException("Задача не найдена!")
